.pre-commit-config.yaml

```diff
@@ -1,7 +1,12 @@
 ---
 minimum_pre_commit_version: 3.5.0
-default_install_hook_types: [commit-msg, pre-commit, pre-push]
-default_stages: [pre-commit, pre-push]
+default_install_hook_types:
+  - commit-msg
+  - pre-commit
+  - pre-push
+default_stages:
+  - pre-commit
+  - pre-push
```

Comment:
these changes were executed automatically by the pre-commit hook after adding the line

https://github.com/OpenZeppelin/openzeppelin-relayer/pull/236/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R38